0collections/geonames_proxy.js

```diff
@@ -1,6 +1,11 @@
 this.grits = this.grits || {};
 
 Meteor.startup(function() {
+    if (process.env.MONGO_URL) {
+        var mongo_url = process.env.MONGO_URL + '/geonames';
```

Comment:
@frosario the MONGO_URL environment variable has a database path already, so appending one like this probably won't work.